js/main.js

```diff
@@ -4,30 +4,84 @@ var $input = document.querySelector('#photo');
 var $img = document.querySelector('.update-img');
 var $form = document.querySelector('form');
 var $imgSrc = './images/placeholder-image-square.jpg';
+var $ul = document.querySelector('ul');
 
 function handleEntry(event) {
   if ($input.value === '') {
     $img.src = $imgSrc;
   } else {
     $img.src = $input.value;
   }
-
 }
 $input.addEventListener('input', handleEntry);
 
 function handleSubmit(event) {
   event.preventDefault();
 
   var obj = {
-    name: $form.name.value,
+    title: $form.title.value,
     url: $form.url.value,
     notes: $form.notes.value,
     nextEntryId: data.nextEntryId++
   };
   data.entries.unshift(obj);
   $img.src = $imgSrc;
 
+  $ul.prepend(createEntry(obj));
   $form.reset();
 }
 
 $form.addEventListener('submit', handleSubmit);
+
+function createEntry(entry) {
+  var li = document.createElement('li');
+  var row = document.createElement('div');
+  row.className = ('row');
+  li.appendChild(row);
+
+  var half = document.createElement('div');
+  half.className = ('column-half');
+  row.append(half);
+
+  var img = document.createElement('img');
+  img.setAttribute('src', entry.url);
+  half.appendChild(img);
+
+  var otherHalf = document.createElement('div');
+  otherHalf.className = ('column-half');
+  row.append(otherHalf);
+
+  var title = document.createElement('h3');
+  title.textContent = entry.title;
+  otherHalf.appendChild(title);
+
+  var p = document.createElement('p');
+  p.textContent = entry.notes;
+  otherHalf.appendChild(p);
+
+  return li;
+}
+window.addEventListener('DOMContentLoaded', function () {
+  for (var i = 0; i < data.entries.length; i++) {
+    var newLi = createEntry(data.entries[i]);
+
+    $ul.appendChild(newLi);
+  }
+});
+
+var $view = document.querySelectorAll('.view');
+
+function handleContainer(view) {
+  if (!event.target.matches('.click')) {
+    return;
+  }
```

Comment:
This function isn't meant to be an event handler anymore, it's meant to be a function that can be called any time a view needs to be switched. You should make an additional function that can be an event handler for switching between views when an element with a `data-view` attribute is clicked. That function will end up calling this function (which should be renamed to something like `switchView`), and pass the value of the `data-view` attribute as an argument.